tools/releaser/breaking_changes/v20230201002.md

```diff
@@ -0,0 +1,15 @@
+## Breaking Changes
```

Comment:
How is this file generated for a new release? Is this a manual process?

Reply:
Yes. Establishing breaking changes is the only manual process. 
It is based on exemptions in openapi.